_episodes/12.md

```diff
@@ -0,0 +1,263 @@
+   * Possible, but a hack really
+ * PyJDBC 
+   * Wraps DB API around any JDBC driver
+   * Kind of a hack since it goes through JDBC to HTTP, when Trino python
+   client does the same more directly
+ * PyODBC 
+   * Similar hack to PyJDBC
+ * Potentially also possible to talk to via HTTP directly
+   * That’s like reimplementing the trino-python-client
+   * Also see question of the week later
+
+Beyond that, it will vary from application to application.
+
+Let’s find out from our guests how this hangs together in Apache Superset, since
+it is using Python.
+
+## PR of the week: Superset PR 13105 feat: first step native support Trino
+
+In this week's pull request <https://github.com/apache/superset/pull/13105> that
+was graciously added by [dungdm93](https://github.com/dungdm93).
+
+The first thing we need to understand about this addition is the concept of a
+database engine in Superset. A database engine handles a lot of the custom
+interactions between various databases and maps them to the interface that 
+Superset understands. If certain concepts are missing in a certain database, 
+like time granularity or SQL syntax, the database engine for that database
+will indicate to Superset that this is not available, and it won't show up as an
+option or report a concise error to the user. By default, database engines use
+the [base.py](https://github.com/apache/superset/blob/master/superset/db_engine_specs/base.py)
+methods, but each engine, like Trino, will add the custom mappings via the
+[trino.py engine](https://github.com/apache/superset/blob/master/superset/db_engine_specs/trino.py)
+to verify that Superset and Trino have valid interactions.
+
+The main contributions in this pull request was adding a few custom changes for 
+Trino to work correctly with Superset. The main change applied was truncating
+the complex timestamps coming from Trino to a format that Superset is able to
+support during time aggregation operations.
+
+This along with some tests opened up a vast amount of functionality that we're
+now able to accomplish with Trino + Superset. We wanted to feature this because
+it goes to show how a small code change, even one that is not in the Trino
+repository, can have a vast effect on those using Superset and Trino.
+
+Thank you so much to [dungdm93](https://github.com/dungdm93) for making this
+change and further linking Trino into a fantastic project like [Apache
+ Superset](https://superset.apache.org/)!
+
+
+## PR Demo: Superset PR 13105 feat: first step native support Trino
+
+To put this PR to the test, we need to connect Superset to Trino as our
+datasource. 
+
+First, you need to follow [these instructions](https://superset.apache.org/docs/installation/installing-superset-using-docker-compose)
+to install Docker (if you don't already have it installed), clone the superset.
+
+`$ git clone https://github.com/apache/superset.git`
+
+Once you have cloned the repository, before you run docker-compose, you want to
```

Comment:
Next, you need to set up the database driver for Trino. Navigate to the root ...